content/en/metrics/custom_metrics/historical_metrics.md

```diff
@@ -44,13 +44,14 @@ To enable the ingestion of historical metrics for a specific metric:
 
 ### Bulk configuration for multiple metrics
 
-You can enable Historical Metrics Ingestion for multiple metrics at once, rather than having to configure each one individually.
+You can enable or disable Historical Metrics Ingestion for multiple metrics at once, rather than having to configure each one individually.
 
-1.  Navigate to the [Metrics Summary Page][1] and click the **Configure Metrics** dropdown.
+1. Navigate to the [Metrics Summary Page][1] and click the **Configure Metrics** dropdown.
```

Comment:
Hmm.. clicking this link at least in the preview for this PR leads me the metrics types public docs instead of the metrics summary page -- can we make sure it leads to app.datadoghq.com/metric/summary

Reply:
Updated in [129fb8a](https://github.com/DataDog/documentation/pull/25714/commits/129fb8ace15a6c2374e88cc023e6420ea90a870f)